### DISCUSSION TITLE
Review integration tests

### DISCUSSION THREAD

**Discussion by mtitov** [radical-cybertools/radical.pilot#3005] [General]
- Start with MPI-RAPTOR on Summit

**Comment by mturilli** [comment]
* Second use case: PSI/J on Summit

**Comment by mturilli** [comment]
Suggestions:
* Use the testing framework we developed for PSI/J in the context of ExaWorks https://github.com/ExaWorks/psij-testing-service:
  . We would get reporting 'for free'
  . We would run our own instance of the dashboard on three.radical-project
  . Can we use GH actions to report via badges? 

* Whatever we use if different from what used for exaworks, we should contribute back.
* Support use case (UC) MPI-RAPRTOR:
  . Write an example to use with the testing-service of ExaWorks

* UC RP-RAPTOR could be part of the existing tests for Parsl-RP integration on the official ExaWorks test service.

* Deploy our own instance of testing-service, likely privately maintained and not public.
